tests/codeception/acceptance/Content/BasicPageCest.php

```diff
@@ -68,6 +68,7 @@ public function testH1Tags(AcceptanceTester $I) {
   public function testRevisionPage(AcceptanceTester $I) {
     $I->logInWithRole('site_manager');
     $node = $I->createEntity(['title' => 'Foo Bar', 'type' => 'stanford_page']);
+    $I->runDrush('cache-rebuild');
```

Comment:
I wonder... instead of doing a full cache rebuild, what if you modify the `createEntity` method in [stanford-caravan](https://github.com/SU-SWS/stanford-caravan/blob/8.x-2.x/src/Codeception/Drupal/DrupalEntity.php#L90). Try adding something like `\Drupal::entityTypeManager()->getStorage($type)->resetCache();`